include/mixer.h

```diff
@@ -244,9 +246,16 @@ class MixerChannel {
 	} zoh_upsampler = {};
 
 	struct {
-		FilterState state = FilterState::Off;
-		std::array<Iir::Butterworth::LowPass<max_filter_order>, 2> lpf = {};
-	} filter = {};
+		struct {
+			FilterState state = FilterState::Off;
+			std::array<Iir::Butterworth::HighPass<max_filter_order>, 2> hpf = {};
+		} highpass = {};
+
+		struct {
+			FilterState state = FilterState::Off;
+			std::array<Iir::Butterworth::LowPass<max_filter_order>, 2> lpf = {};
+		} lowpass = {};
```

Comment:
- highpass.hpf
 - lowpass.lpf 

The sub-names feel like redundant when compared to the parent name.
(we could abbreviate the two `.state`s to be: highpass.hps and lowpass.lps`?)
 
Maybe just `.filter` for both? 
 - highpass.filter
 - lowpass.filter

Consistent w/ using `.state` equally in both:
 - highpass.state
 - lowpass.state

(ok.. enough :bike: :shed:'ing :sweat_smile: )

Reply:
It's a kind of an aesthetic choice, I like this better

```cpp
for (auto &f : filters.highpass.hpf)
    f.setup(order, mixer.sample_rate, cutoff_freq);
```

than

```cpp
for (auto &f : filters.highpass.filters)
    f.setup(order, mixer.sample_rate, cutoff_freq);
```

Do you have any better suggestion than using filters for a second time? I can't think of any good name...